src/repository/mysql/resolution.js

```diff
@@ -8,13 +8,19 @@ export default class MySQLResolution {
   }
 
   async add(data) {
-    let expire = null;
+    const resolutionData = {
+      doctor_name: data.doctorName,
+      doctor_specialization: data.doctorSpecialization,
+      patient_id: data.patientId,
+      ttl: data.ttl,
```

Comment:
Looks like this property are not required for this object.

Reply:
`ttl: data.ttl` removed